.github/workflows/test.yaml

```diff
@@ -54,67 +60,23 @@ jobs:
         restore-keys: |
             ${{ runner.os }}-pip-
 
-    - name: Install dependencies
-      run: |
-        python -m pip install --upgrade pip tox
-        tox --notest
-
-    - name: Test
+    - name: Disable IPv6
+      if: matrix.noipv6
       run: |
-        python --version
-        tox -q
-
-
-  # We need a builder without IPv6. This is going to be slower than all the
-  # others, but that's ok.
-  testing_no_ipv6:
-    runs-on: ubuntu-20.04
-    env:
-      DISABLE_IPV6: yes
-      TOXENV: "alldeps-withcov-posix,coverage-prepare,codecov-push,coveralls-push"
-    strategy:
-      matrix:
-          python-version: [3.6]
-
-    steps:
-    - uses: actions/checkout@v2
-
-    - name: Set up Python
-      uses: actions/setup-python@v2
-      with:
-        python-version: ${{ matrix.python-version }}
-
-    - name: Get pip cache dir
-      id: pip-cache
-      run: |
-        echo "::set-output name=dir::$(pip cache dir)"
-
-    - name: pip cache
-      uses: actions/cache@v2
-      with:
-        path: ${{ steps.pip-cache.outputs.dir }}
-        key:
-          ${{ runner.os }}-pip-${{ hashFiles('pyproject.toml', 'setup.py',
-          'setup.cfg', 'tox.ini') }}
-        restore-keys: |
-            ${{ runner.os }}-pip-
-
-    - name: Display Python information
-      run: |
-        python --version
-        python -c "import sys; print(sys.prefix)"
-        python -c "import sys; print(sys.exec_prefix)"
-        python -c "import sys; print(sys.executable)"
-        python -c "import struct; print(struct.calcsize('P') * 8)"
-        python -c "print('\nENVIRONMENT VARIABLES\n=====================\n\n')"
-        python -c "import os; [ print(e,v) for (e,v) in os.environ.items() ]"
+        sudo sysctl -w net.ipv6.conf.all.disable_ipv6=1
+        sudo sysctl -w net.ipv6.conf.default.disable_ipv6=1
+        sudo sysctl -w net.ipv6.conf.lo.disable_ipv6=1
+        ip a
 
+    - uses: twisted/python-info-action@v1
     - name: Install dependencies
+      shell: bash
```

Comment:
Perhaps we'd like this as a global default?

https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#defaultsrun
```yaml
defaults:
  run:
    shell: bash
```